src/backend/src/utils/design-reviews.utils.ts

```diff
@@ -0,0 +1,10 @@
+import { HttpException } from './errors.utils';
+
+export function validateMeetingTimes(nums: number[]): number[] {
+  for (let i = 1; i < nums.length; i++) {
+    if (nums[i] !== nums[i - 1] + 1 || !(nums[i] >= 0) || !(nums[i] <= 48)) {
```

Comment:
also could you make this `nums[i] < 0 || nums[i] > 48` to make it slightly more readable